cyfra-runtime/src/main/scala/io/computenode/cyfra/runtime/ExecutionHandler.scala

```diff
@@ -0,0 +1,18 @@
+package io.computenode.cyfra.runtime
+
+import io.computenode.cyfra.core.{GExecution, GProgram}
+import io.computenode.cyfra.core.layout.Layout
+
+object ExecutionHandler:
```

Comment:
What is that?

Reply:
Handles `Execution` effect to linearlize, materialize it and ready it for execution.